### DISCUSSION TITLE
How to suppress messages from Tuner Auto LR Find?

### DISCUSSION THREAD

**Discussion by beelze-b** [Lightning-AI/pytorch-lightning#10534] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Hi all,

Is there a way to suppress messages from the auto lr_find (`trainer.tuner.lr_find`) option?

I have followed details from issues like [this one](https://github.com/PyTorchLightning/pytorch-lightning/issues/4462), but I am still left with the following printed to the console.

![image](https://user-images.githubusercontent.com/7582471/141709521-7a526a65-7a4e-4fae-a4d5-2fe0f13ce785.png)

I have done the following to my loggers:

```
  if not verbose: 
          logging.getLogger("lightning").setLevel(logging.ERROR)
          pl_loggers = [ logging.getLogger(name) for name in logging.root.manager.loggerDict if 'lightning' in name ]
  
          for logger in pl_loggers:
              logger.setLevel(logging.ERROR)
```

**Comment by rohitgr7** [comment]
if you are talking about `DATALOADER: 0 VALIDATE RESULTS`, they comes up only when you call `trainer.validate`

**Comment by beelze-b** [comment]
I meant the progress bar and "Finding best initial lr"

I also haven't directly called validate. This is just from the auto lr find.
